apps/designer/app/designer/features/style-panel/search.tsx

```diff
@@ -56,6 +64,10 @@ export const Search = ({ onSearch }: SearchProps) => {
         type="reset"
         aria-label="Reset search"
         className={resetStyle()}
+        css={{
+          transform: "scale(.75)",
```

Comment:
feels wrong to use transform for this,  would setting width/height on the icon directly be a better approach?